chromium_src/chrome/browser/ui/views/file_system_access/file_system_access_restore_permission_bubble_view.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/grit/generated_resources.h"
+
+#undef IDS_PERMISSIONS_BUBBLE_PROMPT
```

Comment:
Overriding every usage is very fragile. If any new usages of IDS_PERMISSIONS_BUBBLE_PROMPT show up in the code they won't be replaced. Why aren't we doing the override in `generated_resources.h`?

Reply:
Great idea, fixed here 94e32ae022e020648194365928f1e1720ac4f3c8